packages/node_modules/@webex/internal-plugin-mercury/src/socket/socket-base.js

```diff
@@ -3,6 +3,12 @@
  */
 
 import {EventEmitter} from 'events';
+
+import {checkRequired} from '@webex/common';
+import {safeSetTimeout} from '@webex/common-timers';
+import {defaults, has, isObject} from 'lodash';
+import uuid from 'uuid';
```

Comment:
eslint check did this for some reason

Reply:
This usually happens based on the import type, but it is strange that it separated `events` from the rest. We can disregard this formatting change as it's not impacting.